packages/batteries/batteries.3.6.0/opam

```diff
@@ -22,6 +22,9 @@ depends: [
   "benchmark" {with-test & >= "1.6"}
   "num"
 ]
+depopts: [
+  "dune" {>= "2.7"}
```

Comment:
```suggestion
  "dune" {>= "2.7"}
```

@kit-ty-kate the bound here should be inverted into a conflict, right?

Reply:
yes